<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecognitionSettingsPane</class>
 <widget class="QWidget" name="RecognitionSettingsPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>384</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="layoutRecognition">
   <item>
    <widget class="QLabel" name="lblOCRLanguages">
     <property name="text">
      <string>OCR languages:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listOCRLanguages">
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::MultiSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutLanguageButtons">
     <item>
      <widget class="QPushButton" name="btnRefreshLanguages">
       <property name="text">
        <string>Refresh language list</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddLanguage">
       <property name="text">
        <string>Install languageâ€¦</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblAfterOCR">
     <property name="text">
      <string>After OCR:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkNotifyOnFinish">
     <property name="text">
      <string>Show notification dialog after OCR completion</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkSoundOnFinish">
     <property name="text">
      <string>Play sound after OCR completion</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutSoundTest">
     <item>
      <widget class="QPushButton" name="btnTestSound">
       <property name="text">
        <string>Test</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderSoundVolume">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>70</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
